### PR TITLE
Adds pytorch files to noxfile

### PR DESCRIPTION
**Pull request by joshdevins**
We added the `pytorch` module which is type checked but was not in the noxfile as such.

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -44,6 +44,9 @@
     "eland/ml/_optional.py",
     "eland/ml/_model_serializer.py",
     "eland/ml/ml_model.py",
+    "eland/ml/pytorch/__ini__.py",
```
Comment (sethmlarson) [inline noxfile.py:47]: Typo in `__init__`

**Review by sethmlarson — APPROVED** [review]
LGTM, one thing to fix though:

File: noxfile.py (hunk 1 above)
Comment (joshdevins) [reply noxfile.py:47]: I just type so fast and the `t` fell off.

**Comment by joshdevins** [comment]
@sethmlarson Can you have a quick pass again? I tried to go through and also fix the type warning.

File: eland/ml/pytorch/transformers.py [hunk 2]
```diff
@@ -390,7 +387,7 @@ def _create_config(self):
                     "max_sequence_length"
                 ] = max_sequence_length
 
-        if self._traceable_model.classification_labels():
+        if isinstance(self._traceable_model, _TraceableClassificationModel):
```
Comment (sethmlarson) [inline eland/ml/pytorch/transformers.py:390]: Are we losing a comparison here? Previously we also called `.classification_labels()`, do we need both or is the `isinstance` enough?

**Review by sethmlarson — APPROVED** [review]
LGTM, had one quick question but barring that we can merge.

File: eland/ml/pytorch/transformers.py (hunk 2 above)
Comment (joshdevins) [reply eland/ml/pytorch/transformers.py]: I was running in to mypy errors but I've fixed that so I've reverted to the previous approach.

**Pull request merged**